File: apps/minecraft/spigot/src/main/java/crabcraft/net/crabUtilities/bingo/BingoDetector.java
package crabcraft.net.crabUtilities.bingo;

import java.util.UUID;
import org.bukkit.event.Listener;

/** Lifecycle shared by the event-driven weekly bingo detector groups. */
public interface BingoDetector extends Listener {
    void resetPlayer(UUID playerId);

    void clear();
}

File: apps/minecraft/spigot/src/main/kotlin/crabcraft/net/crabUtilities/bingo/BingoDetector.kt
package crabcraft.net.crabUtilities.bingo

import java.util.UUID
import org.bukkit.event.Listener

/** Lifecycle shared by the event-driven weekly bingo detector groups. */
interface BingoDetector : Listener {
    fun resetPlayer(playerId: UUID)

    fun clear()
}
